CuratedContent/bssw-tutorials.md

```diff
@@ -0,0 +1,26 @@
+### Better Scientific Software (BSSw) Tutorial
+<!-- deck text start --> 
+
+Keeping skills up to date and focused helps keep work at its best. The BSSw Tutorial page presents a vertical sampler of upcoming and past tutorials on improving scientific software.
+
+#### Contributed by [Deborah Stevens](https://github.com/haikudeb)
+#### Publication date: May 08, 2023
+
+Resource information | Details
+:--- | :--- 
+Article title | [The Better Scientific Software (BSSw) Tutorial](https://bssw-tutorial.github.io)
+Authors | [Deborah Stevens](https://github.com/haikudeb)
+Focus | Software Engineering Tutorials
+Website | https://bssw-tutorial.github.io 
```

Comment:
```suggestion
Resource information | Details 
:--- | :--- 
Tutorial title  | The Better Scientific Software (BSSw) Tutorial 
Presenter | [David Bernhold](https://github.com/bernhold) and the [IDEAS Productivity project](mailto:IDEASProductivity@gmail.com)
Web links | https://bssw-tutorial.github.io/
```

Reply:
@rinkug this is now styled as per [here](https://betterscientificsoftware.github.io/bssw.io/bssw_styling_curated.html) and @bernhold I listed you and ideas productivity project.

Reply:
It really isn't appropriate to call me out here.  Please don't.  I would much prefer it to be just the IDEAS side of that entry.  Alternatively 'The [BSSw Tutorial Team](mailto:bssw-tutorial@lists,mcs.anl.gov)`.

And, for the record, there's a difference between my GH username (bernhold) and my last name (Bernhold**t**).  In my early days, UNIX supported only 8 characters for usernames, and I've stuck with it ever since (wherever I get to choose).

Reply:
I edited the comment and removed you.